echopype/convert/parse_azfp.py

```diff
@@ -216,6 +235,7 @@ def _test_valid_params(params):
         temperature_is_valid = _test_valid_params(["ka", "kb", "kc"])
         tilt_x_is_valid = _test_valid_params(["X_a", "X_b", "X_c"])
         tilt_y_is_valid = _test_valid_params(["Y_a", "Y_b", "Y_c"])
+        pressure_is_valid = _test_valid_params(["a0", "a1"])
```

Comment:
A cosmetic change: move `pressure_is_valid` to be right after `temperature_is_valid`. Temperature and pressure come from external sensors that are sometimes not present (especially pressure). In that sense, I think it's a bit more logical to have those two variables be together.

Reply:
Updated! Moved `pressure_is_valid` check right after `temeperature_is_valid`.